pkg/agent/proxy/proxier.go

```diff
@@ -290,10 +274,10 @@ func (p *proxier) installNodePortService(groupID binding.GroupIDType, svcPort ui
 		svcIP = agentconfig.VirtualNodePortDNATIPv6
 	}
 	if err := p.ofClient.InstallServiceFlows(groupID, svcIP, svcPort, protocol, affinityTimeout, nodeLocalExternal, corev1.ServiceTypeNodePort, false); err != nil {
-		return fmt.Errorf("failed to install Service NodePort load balancing flows: %w", err)
+		return fmt.Errorf("failed to install NodePort load balancing flows: %w", err)
```

Comment:
This patch is actually error-prone as far as I can tell, hope it could introduce less unrelated change to make reviewers focus on the logic change. So do other PRs.

Reply:
Sorry, thinks for reminding, I got that.